Library/Formula/node.rb

```diff
@@ -25,6 +25,7 @@ class Node < Formula
   option "without-completion", "npm bash completion will not be installed"
 
   depends_on :python => :build
+  depends_on :xcode => :build
```

Comment:
[This thread](https://github.com/npm/npm/issues/7032#issuecomment-68354376). It goes off-topic towards the end.